### DISCUSSION TITLE
Trouble getting HTTP access logs working in

### DISCUSSION THREAD

**Discussion by spkane** [linkerd/linkerd2#9704] [Help] [answered]
Hi. I am looking for some help getting the HTTP access logs working.

I am just getting started with `linkerd` and have installed the CRDs (_linkerd-crds 1.4.0_) and the control-plane  (_linkerd-control-plane 1.9.3_) components via helm. I then created a namespace and annotated it with:

```
    linkerd.io/inject: enabled
    config.linkerd.io/access-log: "json"
    config.linkerd.io/proxy-log-level: "warn"
```

I then deployed a pod into that namespace that is running a single container, which provides a very simply HTTP/HTTPS based service.

In general it seems to work and I can see the proxy-logs (when I turn the level up to `info`), but I do not see any HTTP access-logs for my service, even though I can run `kubectl port-forward` and connect to the pod's service from my local browser.

I am using:
  * `cr.l5d.io/linkerd/controller:stable-2.12.1`

Any ideas what I am doing wrong here? Am I side-stepping the proxy with the port-forward, or something else?

**Comment by alpeb** [comment, marked as answer]
That's right, port-forward goes straight into the container's port, bypassing the proxy. Also remember you have to restart the pod for the namespace annotations to be picked up :wink:

**Comment by spkane** [comment]
@aleb Thank you. I thought this might be the case.

Is there an easy way for me to send traffic through the proxy without a second service or do I need to set that up to generate any HTTP logging?

**Reply by alpeb** [reply]
You'll have to use another service from within the cluster. I usually use cURL from a pod started like this:
```
kubectl run -it --rm --restart Never eraseme --image curlimages/curl sh
```

**Reply by spkane** [reply]
Ok. That makes complete sense. Thanks!
